OG/OG/ASTBuilding/TreeNodes/FunctionNode, FunctionCallNodes and extractors/FunctionCalls/ParameterNodeExtractor.cs

```diff
@@ -157,5 +158,33 @@ public override ParameterNode VisitPassedEndPointReference(OGParser.PassedEndPoi
             PointReferenceNode pointRef = new PointReferenceNodeExtractor(SemanticErrors).ExtractPointReferenceNode(endpointContext);
             return new ParameterNode(new IdNode(context.endpointRef.id.Text), pointRef);
         }
+        
+        private ExpressionNode InferIdType(OGParser.PassedIDContext context)
+        {
+            IToken idContext = context?.id;
+          
+            if (idContext != null)
+            {
+                switch (idContext.Text)
+                {
+                    case "number":
+                        return new MathIdNode(idContext.Text, new IdNode(idContext.Text));
+                    case "bool":
+                        return new BoolExprIdNode(idContext.Text, new IdNode(idContext.Text), BoolNode.BoolType.IdValueNode);
+                    case "point":
```

Comment:
Generelt bryder jeg mig ikke om "magiske variable", ie.  burde "point" være POINT sat som en form for global konstant.
Men kan se igennem fingre med det her ;)

Reply:
Ja Rumle har også været lidt træt af min naive tilgang til dette. og jeg er selvfølgelig eninig, dette var blot det tape jeg lige havde. det er gennemgående at hele symboltable er strenge baseret